pkg/internal/flags.go

```diff
@@ -18,3 +18,4 @@ package internal
 
 // Used by tests to selectively disable experimental JSON unmarshaler
 var UseOptimizedJSONUnmarshaling bool = true
+var UseOptimizedJSONUnmarshalingV3 bool = false
```

Comment:
@apelisse Added an additional flag for v3 unmarshalling. This should make it easier to differentiate the performance improvement between

1. No optimization
2. Optimization only in the swagger schema (already committed as part of v2)
3. Optimization with the full openapi v3 spec